lib/reline/config.rb

```diff
@@ -173,7 +173,7 @@ def read_lines(lines, file = nil)
 
       no += 1
 
-      line = line.chomp.lstrip
+      line = line.gsub(/#.*$/, '').chomp.strip
```

Comment:
Sorry for making confusion, I think we don't need to explicitly implement inline comment because readline does not have it.
But we need to retain `set editing-mode vi # pseudo inline comment` ignoring `" # pseudo inline comment"` part because readline ignores it and act like inline comment exist in some directives.
It is already implemented in L186
```ruby
# value ignores " # pseudo comment" part. raw_value does not.
var, value, raw_value = $1.downcase, $2.partition(' ').first, $2
```

Reply:
> because readline does not have it.

Sorry for not getting the whole picture 💦 Removed this change!